### DISCUSSION TITLE
streaming MPD audio output over internet to remote device (into browser?)

### DISCUSSION THREAD

**Discussion by bongoo1** [MusicPlayerDaemon/MPD#2160] [Q&A]
i'm using MPD on a RPi for years to play music to a locally attached USB soundcard.
now i wonder if there is a way to output the music over the internet to either a mobile device (e.g. android phone) or to a windows PC (play in the browser), which is in a remote location.
i.e. while the music storage is at home, the audio output goes to a device located elsewhere.

**Comment by kingosticks** [comment]
Like https://mpd.readthedocs.io/en/latest/plugins.html#httpd  ?

**Comment by bongoo1** [comment]
might be, but i'm not sure if this will exactly do it.
is there somewhere an example on how to configure...with some description on the details for better understanding?

**Reply by kingosticks** [reply]
The docs page has a full description of the options and what they do. Http streaming audio is a pretty standard concept with lots of info online. The user experience is akin to a controllable personal online radio station, with a frustrating high-latency control loop. I suggest you try it, see if it fails to meet your needs, and then update your question with the extra requirements.

You can see the other output plugins on there. You could also try the Snapcast one, not sure how great that'll be over the internet.

**Comment by MaxKellermann** [comment]
This is not a use case MPD was designed for. It can be done with streaming, as @kingosticks wrote, but it's a lousy experience.
